app-web/openlist/autobuild/beyond

```
@@ -0,0 +1 @@
+ln -svf OpenList "$PKGDIR"/usr/bin/openlist
```

Add abinfo

---

Also drop the `-f` flag as it silences errors (on overwrite).